tasks/verify-enterprise-contract/0.1/verify-enterprise-contract.yaml

```diff
@@ -112,6 +117,22 @@ spec:
       command: [ec]
       args:
         - version
+    - name: initialize-tuf
```

Comment:
I was wondering if `when` was available on Steps, seems it is available only on Tasks, would be nice not to have shell code here. Also I wondered if this should be in a sidecar instead.

Reply:
Yeah, `when` on Steps would've been nice. I don't see anything in the code that would support that unfortunately. I think the bash is pretty minimal and there's sufficient coverage in the acceptance tests though. Let's try that out first.

I don't think a sidecar is the right pattern here. My understanding is that a sidecar is meant when you need something to be running throughout the lifecycle of a Task, e.g. setup a proxy that is used in the steps. Here, we just need to initialize TUF root once and early. The Steps interact with it via the filesystem which does already persist throughout the Task lifecycle.